docs/source/admin/quick_howto/anonymous_blocking.rst

```diff
@@ -33,9 +33,9 @@ Configure Anonymous Blocking
 			"name": "Anonymous IP Blocking Policy",
 
 			"anonymousIp": { "blockAnonymousVPN": true,
-			                 "blockHostingProvider": true,
-			                 "blockPublicProxy": true,
-			                 "blockTorExitNode": true},
+				"blockHostingProvider": true,
+				"blockPublicProxy": true,
+				"blockTorExitNode": true},
```

Comment:
This was actually alignment, not indentation. The intent is to line up all the text - and that's totally valid because the example block is properly indented so the contents are not interpreted by the RST compiler at all (they are lex'd by Pygments but this is lexically and syntactically correct JSON so that doesn't matter either).

It is more consistent to indent these rather than align them, but if you're gonna do that I'd ask that you also indent the top property (`"blockAnonymousVPN"`) so that everything still lines up, and also check the output to be sure it isn't being expanded wider than a typical browser window (because Sphinx auto-expands tabs in code examples to GNU-standard 8 spaces and I've already tried everything I can think of to fix that).

Reply:
Reverted in 0220c7acba.